### ISSUE TITLE
ansible-vault decrypt does not print to stdout

### ISSUE THREAD

**Issue by tf42src** [ansible/ansible#31764]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
ansible-vault

##### ANSIBLE VERSION
```
ansible-vault 2.4.0.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['<home_dir>/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /opt/envs/ansible/lib/python3.6/site-packages/ansible
  executable location = /opt/envs/ansible/bin/ansible-vault
  python version = 3.6.1 (default, Apr 22 2017, 00:00:00) [GCC 4.8.4]
```

##### CONFIGURATION
No changes to config

##### OS / ENVIRONMENT
Ubuntu 14.04

##### SUMMARY
ansible-vault decrypt does not print to stdout

##### STEPS TO REPRODUCE
```
> echo "hello" > /tmp/foo
> ansible-vault encrypt /tmp/foo
Encryption successful
> ansible-vault decrypt /tmp/foo --output -
Decryption successful
```

##### EXPECTED RESULTS
```
hello
Decryption successful
```

##### ACTUAL RESULTS
```
Decryption successful
```

**ansible-vault==2.3.1.0 yields the expected output.**

_Additional question: Is it possible to suppress "Decryption successful" as I would like to use the output in a script._

**Comment by alikins** [comment]
This is a duplicate of https://github.com/ansible/ansible/issues/30550

https://github.com/ansible/ansible/pull/31066 has the fix that will be in the 2.4.1.0 release (via https://github.com/alikins/ansible/commit/576e3378a9ef20d96da6a2161141a99a521db6a1)

duplicate_of: 30550

**Comment by tf42src** [comment]
My bad. Searched but couldn't find it.
